arch/arm64/configs/xiaomi-tissot.config

```diff
@@ -1,9 +1,11 @@
 CONFIG_BACKLIGHT_QCOM_WLED=y
 CONFIG_BATTERY_QCOM_FG=m
+CONFIG_CHARGER_QCOM_SMBCHG=m
```

Comment:
to make them work I  had to set smchg and fg to =y

Reply:
In case of ltrf216a don't forget to add `CONFIG_LTRF216A`

Reply:
See comments from @fekz115. Let @alikates judge us

Reply:
Yes, they can be builtin or module. Both have to be the same, but now OTG and role detection works, the usb driver depends on it too and iirc that one is builtin.

Reply:
> Yes, they can be builtin or module. Both have to be the same, but now that OTG and role detection works, the usb driver depends on it too and iirc that one is builtin.

So what do I change to build-in?

Reply:
CONFIG_BATTERY_QCOM_FG=y
CONFIG_CHARGER_QCOM_SMBCHG=y